enterprise/admin/views.py

```diff
@@ -199,8 +200,19 @@ def _enroll_users(cls, emails, course_details, mode, request):
             ).format(enrolled_count=enrolled_count, course_id=course_id))
         if non_existing:
             messages.warning(request, _(
-                "The following users do not have accounts yet and were not enrolled: {}"
+                "The following users do not have accounts yet and were not yet enrolled: {}"
```

Comment:
@mattdrayer They can have the cone, just not the ice cream. Unless they have to eat gluten-free, in which case they're outta luck.

The following users do not have an account on `<platform name>`. They have not been enrolled in the course. When these users create an account, they will be enrolled in the course automatically.